packages/js/product-editor/src/components/attribute-control/attribute-table-row.tsx

```diff
@@ -29,6 +30,38 @@ interface FormTokenFieldProps extends CoreFormTokenField.Props {
 const FormTokenField =
 	CoreFormTokenField as React.ComponentType< FormTokenFieldProps >;
 
+/*
+ * Type copied form core FormTokenField component.
+ * Todo: move to a shared location.
+ */
+export interface TokenItem {
```

Comment:
For now, since this isn't used outside of this file, and you plan on moving it to another location, we don't need to export the interface.